README.developer

```diff
@@ -46,6 +46,29 @@ command can be run within a sub directory to only run a particular set of
 tests (although you may experience issues with this method, running from the
 root ola directory is guaranteed to work).
 
+Branches, Versioning & Releases
+-------------------------------
+
+Version numbers take the form MAJOR.MINOR.PATCH. The release lifecycle is:
+
+- New feature work occurs on the master branch.
+- Once the new features are considered stable or enough time has passed, a new
+  minor release branch will be created, e.g. 0.9.
+- The minor release branch will be stablized with bugfixes, these bug fixes
+  will be merged back into master.
+- Once declared stable, a new patch branch 0 will be created e.g. 0.9.0
```

Comment:
Can't the patch release numbers just be done with commits and tags to the minor release branch, as we shouldn't make further commits to them/they're going to get merged back into minor anyway?